src/Resources/skeleton/query/QueryHandler.tpl.php

```diff
@@ -0,0 +1,20 @@
+<?= "<?php\n" ?>
+
+namespace <?= $namespace; ?>;
+
+<?= $use_statements ?>
+
+#[AsMessageHandler]
+class <?= $class_name ?> implements QueryHandler
+{
+    /**
+    */
+    public function __construct()
+    {
+    }
```

Comment:
**suggestion**: Probably not needed to include it here, since it's entirely optional.